docs/docs/api/components/keyboard-toolbar/index.mdx

```diff
@@ -36,6 +36,7 @@ import toolbar from "./toolbar.lottie.json";
 - **Supports dark/light theme** 🌓: Adapts to the theme settings of the user's device for a seamless experience.
 - **Extended accessibility support** 🔍: Ensures that all users, including those with disabilities, can navigate through inputs effectively.
 - **Full control over the buttons behavior** 🔧: Customize the actions triggered by the next, previous, and done buttons according to your needs.
+- **Extends ViewProps** 📜: Supports all the props that `View` component has.
 
 ## Props
 
```

Comment:
Can we add it here something like:

```
### View Props

Inherits [View Props](https://reactnative.dev/docs/view#props).
```

RN website uses such style, for example in: https://reactnative.dev/docs/keyboardavoidingview#view-props

Reply:
For transparency: I'd like to keep `**Extends ViewProps** 📜: Supports all the props that `View` component has.` text, but also would like to add a link to ViewProps (in the same way as official RN docs do)